modules/registry-configuring-storage-azure-gov-cloud.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+//* registry/configuring_registry_storage-azure.adoc
+
+[id="registry-configuring-storage-azure-gov-cloud_{context}"]
+= Configuring registry storage for Azure Government Cloud
+
+During installation, your cloud credentials are sufficient to create Azure Blob
+Storage, and the Registry Operator automatically configures storage.
+
+.Prerequisites
+
+* A cluster on Azure with Government Cloud infrastructure.
```

Comment:
`A cluster on Azure with user-provisioned infrastructure in a government region`?